ios/xcode/FluentUILib_common.xcconfig

```diff
@@ -9,3 +9,7 @@ OTHER_LDFLAGS = -ObjC
 PRODUCT_NAME = FluentUI
 SKIP_INSTALL = YES
 SWIFT_VERSION = 5.0
+
+// Objective C header file location paths
+HSP_INPUT = $(DERIVED_SOURCES_DIR)
```

Comment:
update the script itself as well? also can we also add FLUENT as prefix of these alias?